dbt/include/oracle/macros/materializations/seed/seed.sql

```diff
@@ -7,17 +8,20 @@
     {% for chunk in agate_table.rows | batch(batch_size) %}
         {% set bindings = [] %}
 
+        {% for row in chunk %}
+            {% do bindings.extend(row) %}
+        {% endfor %}
+
         {% set sql %}
-            insert into {{ this.render() }} ({{ cols_sql }})
+            insert all
             {% for row in chunk -%}
-                select
+              into {{ this.render() }} ({{ cols_sql }}) values(
                 {%- for column in agate_table.column_names -%}
-                    {{" '" + row[loop.index - 1] + "' " }}
+                    :p{{ loop.index }} 
```

Comment:
I have the same question I had in your other PR. Is this ':p' a jinja feature? Is it a typo?

Reply:
Hi @vitoravancini , no, this is how you define binded variables in a parametrized statement (I did not know that a few weeks ago)

[](https://cx-oracle.readthedocs.io/en/latest/user_guide/bind.html)

Reply:
Ah allright!